pkg/rbd/nodeserver.go

```diff
@@ -227,17 +281,11 @@ func (ns *NodeServer) createTargetPath(targetPath string, isBlock bool) (bool, e
 // NodeUnpublishVolume unmounts the volume from the target path
 func (ns *NodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpublishVolumeRequest) (*csi.NodeUnpublishVolumeResponse, error) {
 	targetPath := req.GetTargetPath()
```

Comment:
Move this line below the validate call below, as validation is ensuring target path is non-empty.